### ISSUE TITLE
Dynamically skipping a test causes extra blank log output

### ISSUE THREAD

**Issue by FFdhorkin** [archfz/cypress-terminal-report#99]
cypress 5.6.0
in plugins: installLogsPrinter(on, { printLogsToConsole: 'onFail', includeSuccessfulHookLogs: false });
in support: installLogsPrinter({ enableExtendedCollector: true });

Reproable spec:
```
describe('Nested log Test', () => {
    afterEach(() => {
    });

    it('test1', () => {
    });

    it('test2', () => {
    });

    it('test3', () => {
        cy.state('runnable').ctx.skip();
    });

    it('test4', () => {
    });
});
```

With cypress-terminal-report (left) vs without (right):
![image](https://user-images.githubusercontent.com/59899746/118013263-c9198280-b306-11eb-9864-eb525976cdb0.png)

**Comment by archfz** [comment]
Fix released in 3.1.2.

**Comment by FFdhorkin** [comment]
Confirmed fix :) Thank you!
